scripts/sct_straighten_spinalcord.py

```diff
@@ -87,13 +87,13 @@ def smooth_centerline(fname_centerline, algo_fitting='hanning', type_window='han
         indices_to_remove = []
 
         # ascending verification
-        for iz in range(0, nz_nonz / 2, 1):
+        for iz in range(0, nz_nonz // 2, 1):
```

Comment:
are we 100% sure it does the same thing? if not, it will be hard to debug...

Reply:
The integer division does the same in python2.7 and 3.